api/Sources/Api/PairQL/MacApp/Resolvers/CheckIn_v2.swift

```diff
@@ -163,9 +163,17 @@ func resolveLatestRelease(
   currentAppVersion: String,
   in db: any Client
 ) async throws -> CheckIn_v2.LatestRelease {
-  let releases = try await Release.query()
-    .orderBy(.semver, .asc)
-    .all(in: db)
+  var query = Release.query().orderBy(.semver, .asc)
+
+  // special case, bug in 2.7.0/1 was fixed by a db change
+  // to screenshot rate, so don't force them to update
+  // but people behind 2.7.x should go up to <=2.7.2
+  // delete next time we ship a version we want all to upgrade to
+  if currentAppVersion == "2.7.0" || currentAppVersion == "2.7.1" {
+    query = query.where(.semver != "2.7.2")
```

Comment:
as grungy as this is, i'm really happy about this code, because in `v2.5.0` i snuck in a change where the mac app sends it's current version to the API when asking if there is an update, and i did it for exactly a scenario like this. now that we have this info, we can "pretend" there is no update for users on specific versions, since releasing an update is annoying, and slightly dangerous for users whose filters don't maintain communication after a replace.